frontend/src/components/LocalhostScanForm/LocalhostScanForm.tsx

```diff
@@ -26,6 +27,7 @@ export const LocalhostScanForm: React.FC<Props> = ({ setSelected, disabled }) =>
   const css = useStyles()
 
   useEffect(() => {
+    Analytics.Instance.track('ScanLocalNetwork')
```

Comment:
Isn't event `scanNetwork`?

Reply:
yeah, should be networkScan. missed the second one. updated